src/main/scala/org/virtuslab/akkasaferserializer/BorerAkkaSerializer.scala

```diff
@@ -6,10 +6,11 @@ import io.bullet.borer.{Cbor, Codec, Decoder, Encoder}
 import java.util.concurrent.atomic._
 import scala.reflect.ClassTag
 
-trait CborAkkaSerializer[Ser] extends Serializer {
+trait BorerAkkaSerializer[Ser] extends Serializer {
 
   private val registrations = new AtomicReference[List[(Class[_], Codec[_])]](List.empty)
 
+  //noinspection UnitMethodIsParameterless
```

Comment:
Hmm what tool was raising this warning? is it b/c of Intellij?

Reply:
It was Intellij warning